projects/packages/my-jetpack/_inc/components/my-jetpack-tab-panel/products-content.tsx

```diff
@@ -1,14 +1,25 @@
 import { __ } from '@wordpress/i18n';
+import UnownedProductsCard from '../product-cards-section/unowned-products-card';
+import styles from './styles.module.scss';
 
 /**
  * The Products content component.
  *
  * @return The rendered component.
  */
-export function ProductsContent() {
+const ProductsContent = () => {
```

Comment:
Why change the function declaration?

IMHO, `function` is more readable than `const` when declaring/defining a function.

Reply:
Thanks for sharing your opinions. 
In my opinion, `const` is more readable and aligns better with React and other files in the project.
It's an interesting point about stack trace, thanks for highlighting it.

I don't find `export` at the end of the file an overhead. It's one place to check for exports in the whole file.

Reply:
> `const` is more readable and aligns better with React.

Oh, I didn't know about the second part. Thanks

Do you mind sharing any resources to learn more about its "better alignment" with React?